Shared/messages/MessageFeedsController.cpp

```diff
@@ -87,6 +87,10 @@ MessageFeedsController::~MessageFeedsController()
 void MessageFeedsController::setGeoView(GeoView* geoView)
 {
   m_geoView = geoView;
+
+  // now that the geoview is ready, setup the feeds
+  if (m_geoView && m_messageFeeds->rowCount() == 0)
```

Comment:
I guess it's possible that there will not be any feeds. But presumably this only gets called once, when the geoview is set up, so we shouldn't get repeated calls to `setupFeeds();`?

Reply:
Correct as far as I know. `m_messageFeeds->rowCount() == 0` was the original check and I've just expanded it to include the geoview check.